openlibrary/templates/lists/home.html

```diff
@@ -30,15 +30,16 @@ <h2>$_('Active Lists')</h2>
     <!-- FOR EACH LIST -->
     $for list in lists:
         <div class="listEntry">
-            <div class="cover">
```

Comment:
Could you summarize in the pull request description why we needed to change the HTML markup here to address an accessibility issue?

Reply:
Updated

Reply:
thank you